certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,918 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SMART GLOBAL GOVERNANCE",
+        "description": "Smart Global Governance allows to interact with Smart Compliance Booster solution",
+        "version": "1.0",
+        "contact": {
+            "name": "Benoit Guignard",
```

Comment:
This contact information will be visible to the public, so we do not recommend having PII here. Instead, we would recommend using more generic contact information, such as your support team.